Dockerfile

```diff
@@ -32,6 +20,36 @@ RUN pip3 install -e git+https://github.com/ckan/ckanext-googleanalytics.git#egg=
 RUN pip3 install -e git+https://github.com/ckan/ckanext-showcase.git#egg=ckanext-showcase && \
     pip3 install -r ${APP_DIR}/src/ckanext-showcase/requirements.txt
 
+# Development stage
+FROM ckan/ckan-dev:2.10.4 as dev
+ENV PYTHONPATH=/usr/lib/python3.10/site-packages/
+ENV APP_DIR=/srv/app
+
+# Copy the base extensions installation from the base image
+COPY --from=base ${PYTHONPATH} ${PYTHONPATH}/
+COPY --from=base ${APP_DIR} ${APP_DIR}/
+
+RUN chown -R ckan:ckan ${APP_DIR}
+
+# Clone the extension(s) you are writing for your own project in the `src` folder
+# to get them mounted in this image at runtime
+COPY ckanext-openafrica/ ${APP_DIR}/src/ckanext-openafrica
+RUN cd ${APP_DIR}/src/ckanext-openafrica && python3 setup.py develop
+
+COPY ckanext-datarequests/ ${APP_DIR}/src/ckanext-datarequests
+RUN cd ${APP_DIR}/src/ckanext-datarequests && python3 setup.py develop
+
+# Copy custom initialization scripts for development
+COPY contrib/ckan/docker-entrypoint.d/* /docker-entrypoint.d/
```

Comment:
This one also, shouldn't it be define via docker compose which wouldn't require rebuilding image on change?